### ISSUE TITLE
test_evaluation_parameter_store.py throws errors

### ISSUE THREAD

**Issue by abegong** [fivetran/great_expectations#1150]
Two of the tests in `tests/data_context/store/test_evaluation_parameter_store.py` throw errors on my machine.

Following the instructions on CONTRIBUTING.md, I've executed these steps from the root of the Great Expectations repo:
```
pip install -r requirements-dev.txt 
pip install -e .
pytest —no-spark —no-sqlalchemy
```

All tests pass or skip as expected, except for two in `tests/data_context/store/test_evaluation_parameter_store.py`.

The relevant trace seems to be:
```
/Users/abe/Documents/superconductive/tools/great_expectations/tests/data_context/store/test_evaluation_parameter_store.py:39: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/util.py:116: in instantiate_class_from_config
    class_instance = class_(**config_with_defaults)
/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/store/metric_store.py:57: in __init__
    super(EvaluationParameterStore, self).__init__(store_backend=store_backend)
/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/store/metric_store.py:31: in __init__
    super(MetricStore, self).__init__(store_backend=store_backend)
/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/store/store.py:36: in __init__
    "module_name": "great_expectations.data_context.store"
/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/util.py:116: in instantiate_class_from_config
    class_instance = class_(**config_with_defaults)
/Users/abe/Documents/superconductive/tools/great_expectations/great_expectations/data_context/store/database_store_backend.py:43: in __init__
    meta.create_all(self.engine)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/sql/schema.py:4294: in create_all
    ddl.SchemaGenerator, self, checkfirst=checkfirst, tables=tables
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/base.py:2035: in _run_visitor
    with self._optional_conn_ctx_manager(connection) as conn:
/Users/abe/anaconda2/envs/py3/lib/python3.7/contextlib.py:112: in __enter__
    return next(self.gen)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/base.py:2027: in _optional_conn_ctx_manager
    with self._contextual_connect() as conn:
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/base.py:2229: in _contextual_connect
    self._wrap_pool_connect(self.pool.connect, None),
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/base.py:2269: in _wrap_pool_connect
    e, dialect, self
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/base.py:1536: in _handle_dbapi_exception_noconnection
    util.raise_from_cause(sqlalchemy_exception, exc_info)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/util/compat.py:398: in raise_from_cause
    reraise(type(exception), exception, tb=exc_tb, cause=cause)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/util/compat.py:152: in reraise
    raise value.with_traceback(tb)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/base.py:2265: in _wrap_pool_connect
    return fn()
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/base.py:363: in connect
    return _ConnectionFairy._checkout(self)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/base.py:760: in _checkout
    fairy = _ConnectionRecord.checkout(pool)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/base.py:492: in checkout
    rec = pool._do_get()
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py:139: in _do_get
    self._dec_overflow()
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/util/langhelpers.py:68: in __exit__
    compat.reraise(exc_type, exc_value, exc_tb)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/util/compat.py:153: in reraise
    raise value
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/impl.py:136: in _do_get
    return self._create_connection()
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/base.py:308: in _create_connection
    return _ConnectionRecord(self)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/base.py:437: in __init__
    self.__connect(first_connect_check=True)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/pool/base.py:639: in __connect
    connection = pool._invoke_creator(self)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/strategies.py:114: in connect
    return dialect.connect(*cargs, **cparams)
/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/sqlalchemy/engine/default.py:453: in connect
    return self.dbapi.connect(*cargs, **cparams)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

dsn = 'host=localhost user=postgres port=5432 dbname=test_ci', connection_factory = None, cursor_factory = None, kwargs = {'database': 'test_ci', 'host': 'localhost', 'port': 5432, 'user': 'postgres'}
kwasync = {}

    def connect(dsn=None, connection_factory=None, cursor_factory=None, **kwargs):
        """
        Create a new database connection.
    
        The connection parameters can be specified as a string:
    
            conn = psycopg2.connect("dbname=test user=postgres password=secret")
    
        or using a set of keyword arguments:
    
            conn = psycopg2.connect(database="test", user="postgres", password="secret")
    
        Or as a mix of both. The basic connection parameters are:
    
        - *dbname*: the database name
        - *database*: the database name (only as keyword argument)
        - *user*: user name used to authenticate
        - *password*: password used to authenticate
        - *host*: database host address (defaults to UNIX socket if not provided)
        - *port*: connection port number (defaults to 5432 if not provided)
    
        Using the *connection_factory* parameter a different class or connections
        factory can be specified. It should be a callable object taking a dsn
        argument.
    
        Using the *cursor_factory* parameter, a new default cursor factory will be
        used by cursor().
    
        Using *async*=True an asynchronous connection will be created. *async_* is
        a valid alias (for Python versions where ``async`` is a keyword).
    
        Any other keyword parameter will be passed to the underlying client
        library: the list of supported parameters depends on the library version.
    
        """
        kwasync = {}
        if 'async' in kwargs:
            kwasync['async'] = kwargs.pop('async')
        if 'async_' in kwargs:
            kwasync['async_'] = kwargs.pop('async_')
    
        if dsn is None and not kwargs:
            raise TypeError('missing dsn and no parameters')
    
        dsn = _ext.make_dsn(dsn, **kwargs)
>       conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
E       sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) FATAL:  database "test_ci" does not exist
E       
E       (Background on this error at: http://sqlalche.me/e/e3q8)

/Users/abe/anaconda2/envs/py3/lib/python3.7/site-packages/psycopg2/__init__.py:126: OperationalError
```

I'm not going to try to debug now. Probably a good candidate for a pairing session next week.

**Comment by abegong** [comment]
Looks like this may be related to #818 , but I'm not certain. I thought that running `--no-sqlalchemy` would have the same effect as running `--no-postgresql`

**Comment by spbail** [comment]
> I thought that running `--no-sqlalchemy` would have the same effect as running `--no-postgresql`

+1, that's what was explained to me - might be intended but not fully implemented.

**Comment by jcampbell** [comment]
Fixed via #1199
